### PR TITLE
Add Apple Streaming

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/285

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -258,6 +258,30 @@
...
     },
+    {
+      "id": "apple_streaming",
+      "name": "Apple Streaming",
+      "rules": [
+        "||applemusic.apple^",
+        "||itun.es^",
```
Comment (zloyden) [inline assets/services.json:266]: Does this domain exist? Perhaps you meant `itunes.es`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: assets/services.json (hunk 1 above)
Comment (Sergey-Lyapin) [reply assets/services.json:266]: source:
https://www.netify.ai/resources/applications/apple-itunes

Comment (zloyden) [reply assets/services.json:266]: I see that, but the domain is not valid on my end. What's the purpose of this?
Btw, there is no redirect to `https://www.apple.com/es/itunes/` if you use `itun.es`, only `itunes.es`.

Comment (Sergey-Lyapin) [reply assets/services.json:266]: https://github.com/AdguardTeam/HostlistsRegistry/pull/295/commits/1b152f062c0bad103068f2c13c9edde33fe60911

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**